lib/pronto/bitbucket_server.rb

```diff
@@ -0,0 +1,34 @@
+module Pronto
+  class BitbucketServer < Bitbucket
+    def pull_comments(sha)
+      @comment_cache["#{pull_id}/#{sha}"] ||= begin
+        client.pull_comments(slug, pull_id).map do |comment|
+          if comment['commentAnchor']
+            Comment.new(sha, comment['comment']['text'], comment['commentAnchor']['path'], comment['commentAnchor']['line'])
+          else
+            Comment.new(sha, comment['comment']['text'])
```

Comment:
Bitbucket Server can return not only inline comments, but comments for file and pull request itself, so we need to handle that comments.
I can rewrite this using local variable or use inject instead of map for exclude theirs